data-generator/src/main/java/cloud/project/datagenerator/hotels/HotelsDataGenerator.java

```diff
@@ -55,23 +61,56 @@ private void createNewRoom() {
                 .build();
 
         updateHotelDataInHotelModules(DataUpdateType.CREATE, newRoom);
+
+        updateHotelUpdatesOnFrontend(DataUpdateType.CREATE, newRoom.getName(), randomHotel.getName(), 0, 0);
     }
 
     private void updateRandomRoom() {
-        Hotel randomHotel = getRandomHotel();
-        if (randomHotel == null) return;
+        Pair<Hotel, Room> randomRoomPair = getRandomRoomWithoutReservations();
+        Hotel randomHotel = randomRoomPair.getKey();
+        Room randomRoom = randomRoomPair.getValue();
 
-        Room randomRoom = randomHotel.getRooms().get(random.nextInt(randomHotel.getRooms().size()));
         int currentGuestCapacity = randomRoom.getGuestCapacity();
-        int newGuestCapacity = random.nextInt(1, currentGuestCapacity + 10);
+        int newGuestCapacity = random.nextInt(currentGuestCapacity, currentGuestCapacity + 10);
 
         float currentPricePerAdult = randomRoom.getPricePerAdult();
-        float newPricePerAdult = random.nextFloat(currentPricePerAdult, currentPricePerAdult*10);
+        float newPricePerAdult = random.nextFloat(currentPricePerAdult, currentPricePerAdult * 10);
```

Comment:
parę razy wylosuje ten pokój i cena będzie wywalona w kosmos, lepiej to ograniczyć do np. `+- 500 zł` czy coś

Reply:
Chodziło o to, aby łatwo było zobaczyć zmiany xd. Zmienione: [commit](https://github.com/Microarchitecturovisco/travel-api/pull/57/commits/7ab06a4bfcabb78832364a6f3edb22de585da450)